src/babylon.scene.ts

```diff
@@ -930,6 +930,15 @@
 
         private _depthRenderer: Nullable<DepthRenderer>;
         private _geometryBufferRenderer: Nullable<GeometryBufferRenderer>;
+        public get geometryBufferRenderer(): Nullable<GeometryBufferRenderer> {
+            return this._geometryBufferRenderer;
+        }
+        public set geometryBufferRenderer(geometryBufferRenderer: Nullable<GeometryBufferRenderer>) {
+            this._geometryBufferRenderer = geometryBufferRenderer;
+            if (this._geometryBufferRenderer && !this._geometryBufferRenderer.isSupported) {

```

Comment:
what are you checking here? why not check it before setting the variable?

Reply:
It is somehow the same logic as in enableGeometryBufferRenderer().

Reply:
Well I agree that when we call the setter, we should have already made sure that MRT is supported.

Reply:
you are setting `this._geometryBufferRenderer`, then checking if it is supported, and if not, setting is to null. wouldn't it be better to check for support before setting `this._geometryBufferRenderer`? something like:

```
if (geometryBufferRenderer && geometryBufferRenderer.isSupported) {
    this._geometryBufferRenderer = geometryBufferRenderer;
}
```

Reply:
Totally agree. I will modify it.